bson/src/test/unit/org/bson/DocumentTest.java

```diff
@@ -118,7 +118,7 @@ public void toJsonShouldTakeACustomDocumentCodec() {
             // noop
         }
 
-        assertEquals("{ \"database\" : { \"name\" : \"MongoDB\" } }", customDocument.toJson(customDocumentCodec));
+        assertEquals("{\"database\": {\"name\": \"MongoDB\"}}", customDocument.toJson(customDocumentCodec));
```

Comment:
This change to the formatting is terrible in my opinion.

Reply:
Coming from a JS background having spaces before a `:` and after brackets `{` always felt weird and not a convention I've seen outside MongoDB.

That said the majority of conventions also newlines values in a the dictionary as well as the arrays.